app/models/recurring_todos/abstract_repeat_pattern.rb

```diff
@@ -174,7 +174,7 @@ def continues_recurring?(previous)
     # same day as the previous
     def determine_start(previous, offset=0.day)
       start = self.start_from || NullTime.new
-      now = Time.zone.now
+      now = Time.now
```

Comment:
@dnrce maybe you missed my last comment. Just pinging again to make sure that you see it, since I forgot to mention you before :)